src/components/Dragon.js

```diff
@@ -0,0 +1,38 @@
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { reserveDragon } from '../redux/slices/dragons/drangonsSlice';
+import styles from './Dragons.module.css';
+
+export default function Dragon({
+  id, name, image, reserved, description,
+}) {
+  const dispatch = useDispatch();
+
+  function handleClick(id) {
+    dispatch(reserveDragon(id));
+  }
+
+  return (
+    <li>
+      <figure className={styles.figure}>
+        <img className={styles.image} src={image} alt={`shot of ${name}`} />
+        <div className={styles.details}>
+          <figcaption>{name}</figcaption>
+          <p className={styles.desc}>
+            {reserved && <small className="reserved">Reserved</small>}
+            {description}
+          </p>
+          {reserved ? <button className={styles.btn_cancel} type="button" onClick={() => handleClick(id)}>Cancel Reservation</button> : <button type="button" className={styles.btn} onClick={() => handleClick(id)}>Reserve Rocket</button>}
```

Comment:
- [ ] This is a `Dragon` component not rocket. Kindly chnage the 'Reserve Rocket' to "Reserve Dragon" which is the right naming atched. This will show your keen attention to detail, and also improve your developer skill set for giving high quality code 👍 

## View
![image](https://github.com/mahabubx7/space-cars/assets/50990925/3ab4c160-dabd-430f-9810-bb228e0bebdc)

Reply:
Thank you for this exact information. I will do it asap.